x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.tsx

```diff
@@ -216,15 +271,61 @@ export const StepDefineForm: FC<StepDefineFormProps> = React.memo((props) => {
           </EuiFormRow>
         )}
 
+        {hasValidTimeField && (
+          <EuiFormRow
+            fullWidth
+            label={
+              <>
+                {i18n.translate('xpack.transform.stepDefineForm.datePickerLabel', {
+                  defaultMessage: 'Time range',
+                })}{' '}
+                <EuiIconTip
+                  content={i18n.translate(
+                    'xpack.transform.stepDefineForm.datePickerIconTipContent',
+                    {
+                      defaultMessage:
+                        'The time range will be applied to previews only, it will not be part of the final transform configuration.',
```

Comment:
Suggest editing to `The time range will be applied to previews only and will not be part of the final transform configuration`